containers/azure-static-web-apps/.devcontainer/Dockerfile

```diff
@@ -2,8 +2,22 @@
 # https://github.com/Azure/azure-functions-docker/blob/master/host/3.0/buster/amd64/python/python38/python38-core-tools.Dockerfile
 FROM mcr.microsoft.com/azure-functions/python:3.0-python3.8-core-tools
 
-ENV NVM_DIR="/usr/local/share/nvm"
-ENV NVM_SYMLINK_CURRENT=true \
-    PATH=${NVM_DIR}/current/bin:${PATH}
-COPY library-scripts/node-debian.sh /tmp/library-scripts/
-RUN apt-get update && bash /tmp/library-scripts/node-debian.sh "${NVM_DIR}"
\ No newline at end of file
+# Copy library scripts to execute
+COPY library-scripts/*.sh library-scripts/*.env /tmp/library-scripts/
+
+# Install Node.js and Azure Static Web Apps CLI
+ARG NODE_VERSION="lts/*"
```

Comment:
Does Azure Static Web Apps support Node 16 yet?  That's soon to be LTS.

If there's a static list of supported versions you could add this in the comment to let people pick:
```
# [Choice] Node.js version: 14, 12
```

Reply:
What would you recommend then? I left `lts` as it was there initially, but also wondered about that.
Should we default to Node 12 with a little comment explanation here, and then update the version later when Node 14/16 is supported in SWA?

Reply:
@anthonychu ? 👆 I can merge as "lts/*" if that makes sense... right now that's 14.

Reply:
Yep. I think that'll be okay. Thanks.